virttest/qemu_devices/qdevices.py

```diff
@@ -2430,3 +2497,86 @@ def _update_device_props(self, device, addr):
     def _set_device_props(self, device, addr):
         """ Change value to drive{A,B,...} """
         device.set_param('index', self._addr2stor(addr))
+
+
+class QCPUBus(QSparseBus):
+    """
+    CPU bus representation including cpu/cpu-core handling.
+    """
+
+    def __init__(self, cpu_model, cpu_type, lengths, fixed_vcpu, cpu_topology,
+                 aobject=None, atype=None):
+        """
+        :param cpu_model: cpu model (host/qemu64 ...)
+        :type cpu_model: str
+        :param cpu_type: type of the cpu_model (x86_64-cpu, spapr-cpu-core, ...)
+        :type cpu_type: str
+        :param lengths: number of vcpu device count (maxcpus)
```

Comment:
A little confused by 'vcpus device count' and 'maxcpus'

Reply:
will update